### PR TITLE
Update .travis.yml

### PR DESCRIPTION
**Pull request by greycat001**
- Removing Python 2.7
- Updating Python from 3.5 to 3.6

### CODE REVIEW

**Comment by lioramilbaum** [comment]
@greycat001 please link the issue to this PR

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 language: "python"
 python:
-    - "2.7"
-    - "3.5"
+#    - "2.7"
```
Comment (lioramilbaum) [inline .travis.yml]: remove commented code

**Comment by greycat001** [comment]
Close #116 .

**Comment by greycat001** [comment]
Related to #151.

**Pull request merged**